cirq/protocols/apply_unitary.py

```diff
@@ -79,6 +91,12 @@ def __init__(self,
         self.available_buffer = available_buffer
         self.axes = tuple(axes)
 
+    @staticmethod
+    def default(num_qubits: int) -> 'ApplyUnitaryArgs':
+        """A default instance starting in state |0⟩."""
+        state = linalg.one_hot(shape=(2,) * num_qubits, dtype=np.complex128)
```

Comment:
be explicit here and set index. otherwise I have to dig to understand why this is 0.

Reply:
Done. Not nearly as clean since you need a multi-dimensional index, though.